data/json/mapgen_palettes/house_general_palette.json

```
@@ -149,14 +149,7 @@
       ],
       "r": [ { "item": "office_paper", "chance": 5 }, { "item": "office", "chance": 25 } ],
       "t": { "item": "SUS_toilet", "chance": 75 },
-      "v": [
-        { "item": "art", "chance": 5 },
-        { "item": "home_display_case", "chance": 30 },
-        { "item": "harddrugs", "chance": 10, "repeat": [ 1, 2 ] },
-        { "item": "maps", "chance": 2 },
-        { "item": "guns_pistol_common", "chance": 50, "ammo": 90, "magazine": 100 },
-        { "item": "gemstones", "chance": 30, "repeat": [ 1, 2 ] }
-      ],
+      "v": { "item": "guns_safe", "chance": 100 },
```

Maybe still permit these to spawn fluff items, possibly in addition to calling the gun safe itemgroup?

This already basically eliminates regular safes from houses in favor of only gun safes, so I'd be inclined to keep the fluff in them for now. I have an idea for how regular safes can maybe be brought back as a separate thing, but that can wait for a later PR...
```suggestion
      "v": [
        { "item": "guns_safe", "chance": 100 },
        { "item": "art", "chance": 5 },
        { "item": "home_display_case", "chance": 30 },
        { "item": "harddrugs", "chance": 10, "repeat": [ 1, 2 ] },
        { "item": "maps", "chance": 2 },
        { "item": "gemstones", "chance": 30, "repeat": [ 1, 2 ] }
```